pkgs/tools/misc/kb/default.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
```

```suggestion
, python3
}:

python3.pkgs.buildPythonApplication rec {
```

Outside of python-modules we use python3 directly, since application often require overrides of the python3 packageset.